lib/components/search.js

```diff
@@ -64,7 +64,11 @@ function Searcher(opts) { // jshint -W071
   });
 
   this.controls = this.opts.controls || [];
-
+  
+  if (this.opts.opts.paged===undefined) {
+    this.opts.opts.paged=true;
+  }
```

Comment:
Please include spaces around the operators, e.g. `this.opts.opts.paged === undefined`